packages/scribe/src/editor/Editor.tsx

```diff
@@ -151,7 +187,13 @@ const Editor = forwardRef(function Editor(
       <ContextMenuPlugin />
       <KeyboardShortcutPlugin />
       <ClipboardPlugin />
-      <ScriptureReferencePlugin scrRef={scrRef} setScrRef={setScrRef} />
+      <ScriptureReferencePlugin scrRef={scrRef} onScrRefChange={onScrRefChange} />
+      <ArrowNavigationPlugin />
+      <CommandMenuPlugin />
+      <OnSelectionChangePlugin onChange={onSelectionChange} />
+      <ParaNodePlugin />
+      <TextDirectionPlugin textDirection="auto" />
```

Comment:
If you are going to hard-code the direction as _"auto"_ you probably don't need to use this plugin. The comment for the type `TextDirection` says '_"auto" is included for completeness but it is not expected that "auto" will be of any use for minority languages._'. You really want the user to tell you what direction the minority language is since the browser is not likely to get that right for a language it doesn't know about. On the other hand it will likely to be right more often than not if your user can't tell you.

Reply:
I've made it dynamic with a state variiable.